packages/extras/src/components/BottomSheet.tsx

```diff
@@ -51,14 +51,13 @@ export type BottomSheetProps = {
   overlayStyle?: ViewStyle | ViewStyle[];
   panGestureEnabled?: boolean;
   persistent?: boolean;
-  scrollEnabled?: boolean;
-  scrollViewProps?: Omit<ScrollViewProps, 'scrollEnabled'>;
+  useScrollView?: boolean;
```

Comment:
`use` keyword should be reserved for hooks. We can use it but it may be ambiguous to consumers of the package.

Reply:
happy to use a different name. What would you suggest?